task/host-sp-comms/src/main.rs

```diff
@@ -306,6 +312,12 @@ impl ServerImpl {
                 last_panic: [0; MAX_HOST_FAIL_MESSAGE_LEN],
                 etc_system: [0; MAX_ETC_SYSTEM_LEN],
                 dtrace_conf: [0; MAX_DTRACE_CONF_LEN],
+
+                // Default value for InventoryData
+                scratch: host_sp_messages::InventoryData::DimmSpd {
+                    id: [0u8; 512],
+                    temp_sensor: 0u32,
+                },
```

Comment:
Would there ever be a case where we would see this value in a dump and confuse ourselves? Would it make more sense to add an official "default" value to the inventory?

Reply:
For context, we need to allocate a global static `InventoryData` to (somewhat) reduce stack usage, since it's a gigantic struct.

I'm leaning towards *not* having a default value, but could be persuaded otherwise.

We will see the last value written to `scratch`  in a dump (since it's global), but I hope we won't confuse ourselves with it.  If we added a default value, the host would also need to be aware of it, and we'd have to reset to our default value after reading each inventory item (clearing the ~1 KiB of space).